.github/workflows/test_release_linux.yml

```diff
@@ -0,0 +1,46 @@
+name: Logging Test Linux pipeline
+
+on:
+  push:
+    branches:
+      - '*'
+      - '!master'
+env:
+  TAG: ${{ github.event.release.tag_name }}
```

Comment:
this is not triggered by a release event

Reply:
True, removed from the pipeline!